target/scripts/start-mailserver.sh

```diff
@@ -3,19 +3,22 @@
 # shellcheck source=./helpers/index.sh
 source /usr/local/bin/helpers/index.sh
 
-unset FUNCS_SETUP FUNCS_FIX FUNCS_CHECK FUNCS_MISC
-unset DAEMONS_START HOSTNAME DOMAINNAME
-
-#shellcheck disable=SC2034
+# shellcheck disable=SC2034
 declare -A VARS
 declare -a FUNCS_SETUP FUNCS_FIX FUNCS_CHECK FUNCS_MISC DAEMONS_START
 
+_obtain_hostname_and_domainname
+
 # ------------------------------------------------------------
 # ? <<
 # --
 # ? >> Setup of default and global values / variables
 # ------------------------------------------------------------
 
+VARS[POSTMASTER_ADDRESS]="${POSTMASTER_ADDRESS:=postmaster@${DOMAINNAME}}"
```

Comment:
```suggestion
# These variables must be defined first; They are used as default values for other variables.
VARS[POSTMASTER_ADDRESS]="${POSTMASTER_ADDRESS:=postmaster@${DOMAINNAME}}"
```

Reply:
I honestly believe this to be very obvious, but if other see it as @casperklein, I will add this without further objection.

Reply:
I needed some seconds to realize, why there was a separation to the other variables 😆 I was afraid, that someone else in the future tries to "optimize" it, by removing the empty line and sorting the variables alphabetically.

Reply:
I don't mind either way, I assume the comment was about some of the `VARS[ ... ]` values being defined earlier and used by others below.

Maybe there is value in making that more clear upfront to any new contributors / maintainers that are less familiar, but at the same time, I'd expect breakage or failing tests to catch such dependencies being defined later than they should, or at the very least in review from the core maintainers we have.

Reply:
> I needed some seconds to realize, why there was a separation to the other variables

I had the same reaction initially when first reviewing, but I think that's a one off time.

I don't know if the comment would help much if someone decides to collapse the space separation, would probably be helpful in that case to have a more clear separation (eg: `# ==========` or similar), but that isn't an area that should be getting worked on that often that we'd risk missing such a mistake happening in review?

Reply:
Alright, I will add the comment, but I don't think we need more / extra separators :D